glue/BUILD.gn

```diff
@@ -17,6 +17,11 @@ source_set("glue") {
     "//lib/ftl",
     "//mojo/public/cpp/environment",
     "//mojo/public/cpp/system",
+    "//mojo/services/tracing/cpp",
+  ]
+
+  public_deps = [
+    "//mojo/public/c",
   ]
```

Comment:
Why does this need a public dep on mojo/public/c ?  Perhaps we can use a public dependency on `//mojo/services/tracing/cpp` instead give that is what we're including a public header.